planning/planning_validator/autoware_planning_validator/launch/planning_validator.launch.xml

```diff
@@ -0,0 +1,30 @@
+<launch>
+  <arg name="launch_modules_list_end" default="&quot;&quot;]"/>
+  <arg name="planning_validator_launch_modules" default="["/>
+  <let name="planning_validator_launch_modules" value="$(eval &quot;'$(var planning_validator_launch_modules)' + 'autoware::planning_validator::LatencyValidator, '&quot;)"/>
+  <let name="planning_validator_launch_modules" value="$(eval &quot;'$(var planning_validator_launch_modules)' + 'autoware::planning_validator::TrajectoryValidator, '&quot;)"/>
```

Comment:
(optional) it is currently not possible to enable/disable modules with launch arguments.
To implement this feature some additional arguments need to be defined and used in `if` inside these `<let .../>`.